gen/generate.go

```diff
@@ -76,6 +76,9 @@ type Options struct {
 
 	// Do not generate Zap logging code
 	NoZap bool
+
+	// Name of the file to be generated by thriftRW.
+	SingleFile string
```

Comment:
At this point, seems like this should be "OutputFile"?

Reply:
Changed the name.